modules/Hoot/ui/conflicts/resolve.js

```diff
@@ -82,20 +84,20 @@ export default class Resolve {
      * @param layer - review layer
      */
     performAcceptAll( layer ) {
-        let conflateController = this.sidebar.forms.conflate.controller,
-            key                = {
-                name: layer.name,
-                id: layer.id,
-                color: layer.color
-            };
+        let conflateController = this.sidebar.forms.conflate.controller;
+
+        let loadedLayers = Hoot.layers.loadedLayers;
+
+        _forEach( loadedLayers, lyr => {
+
+            if ( !lyr.isMerged ) {
+                Hoot.layers.removeLoadedLayer( lyr.id );
+            }
+        } );
```

Comment:
This fixes the bug, but introduces another.  By removing the source layers, the "show inputs" button is broken.
![image](https://user-images.githubusercontent.com/2600932/77479691-a53f6880-6df6-11ea-90e9-74ba9c3f2b2a.png)

This makes me wonder what the point of removing the merged layer, only to add it back in might be.  Maybe we don't need to do that at all.

Maybe just forcing a reload of the layer data with review relations updated to resolved?  Would have to look at the http request that Resolve All Remaining makes, it might update the review relations on the back end, then maybe there is another method in iD to force a reload of the tiled layer data.